tests/unit/Transformer/Bug32Cest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+
+namespace Test\Unit\Transformer;
+
+use Diaclone\Resource\ObjectItem;
+use Test\Unit\Support\Entities\Password;
+use Test\Unit\Support\Transformers\PayloadToPasswordTransformer;
+use UnitTester;
+
+class Bug32Cest
```

Comment:
why is it called Bug32Cest?

Reply:
I put in a issue ticket #32 so I just made the test based on the issue. It wasn't a new feature, so I didn't really know what to call it. I'll change it if you have a suggestion.

Reply:
@alexcoffee do you need me to change this?

Reply:
lol no it just made me curious